public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -51,11 +52,17 @@ function getNextRequestId() {
 }
 
 export class PanelQueryRunner {
+  panelId: number; // set by the constructor
```

Comment:
adding private to the constructor and you do not need this (nor setting it in constructor)